### ISSUE TITLE
Moving module sync_mail_forward to OCA

### ISSUE THREAD

**Issue by rafaelbn** [synconics/sync_mail_forward#1]
Hi,

I would like to ask if you are interested in moving this module https://github.com/synconics/sync_mail_forward/tree/master/sync_mail_forward to OCA https://github.com/OCA/social

Thanks
Regards,
Rafael

**Comment by synconics** [comment]
yes, we are ready to do, please let us know the step how we can do that?
